.github/workflows/backend.yml

```diff
@@ -0,0 +1,103 @@
+name: Backend
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - 'release**'
+      - 'feature**'
+      - 'fix**'
+      - 'ci**'
+    tags:
+      - '**'
+    paths:
+    - 'src/metabase/**'
+    - 'modules/drivers/**'
+    - 'enterprise/backend/**'
+    - '**/project.clj'
+    - '**/deps.edn'
+    - '.github/workflows/**'
+
+jobs:
+
+  be-linter-bikeshed:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 10
+    steps:
+    - uses: actions/checkout@v2
+    - name: Prepare JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    - name: Get M2 cache
+      uses: actions/cache@v2
+      with:
+        path: ~/.m2
+        key: ${{ runner.os }}-m2-${{ hashFiles('**/project.clj') }}
+    - run: lein with-profile +ci,+oss bikeshed
+
+  be-linter-eastwood:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 10
+    steps:
+    - uses: actions/checkout@v2
+    - name: Prepare JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    - name: Get M2 cache
+      uses: actions/cache@v2
+      with:
+        path: ~/.m2
+        key: ${{ runner.os }}-m2-${{ hashFiles('**/project.clj') }}
+    - run: lein with-profile +ci,+oss eastwood
```

Comment:
You only need to do `with-profile +ci` for these. They include `with-profile +ee` implicitly, so `+oss` isn't doing anything here and is misleading. We want to include EE code when we run linters either way... just because it's under the EE license doesn't mean we shouldn't lint it

Reply:
Very good point, will make the adjustment accordingly.